.sync.yml

```diff
@@ -1,3 +1,7 @@
 ---
-.travis.yml:
-  forge_password: "HPcnq8CAjz31Lz1Mh4p8QV9orwHoDAaSM73aqhWVYmz21eMCmg4NnyjtIQyR75c3NlDHgDVhEKujT+4WT5V4CvNFIJseD0ox6vwuLwefxJs3jAKYJxDAjCv/sL1GcQwR4IKs+bTh97MBbha2QBPG/REGUYfr3xlVCtxDo2TtnTc="
+Gemfile:
+  optional:
+    ':test':
+      - gem: ruby-augeas
+.github/workflows/ci.yml:
+  additional_packages: libaugeas-dev augeas-tools
```

Comment:
Is there a PR to modulesync_config I missed for this?

Reply:
not yet. I'm not sure how we should properly set the options. Do you have any ideas for https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.github/workflows/ci.yml.erb ?

Reply:
Untested, but https://github.com/voxpupuli/modulesync_config/pull/780 may work.

Reply:
yep, works